### PR TITLE
ko: cr2 Concurrency translations

### PR DESCRIPTION
**Pull request by henrif75**
#262 #1463
*Stats*
Before: 1297 translated messages, 383 fuzzy translations, 1780 untranslated messages.
After: 1493 translated messages, 454 fuzzy translations, 1513 untranslated messages.

**Note**
You can ignore _fuzzy_ translations from sections other than Concurrency.

### CODE REVIEW

**Comment by henrif75** [comment]
Stats after clean-up:
1493 translated messages, 398 fuzzy translations, 1569 untranslated messages.

File: po/ko.po [hunk 1]
```diff
@@ -19459,28 +19494,31 @@ msgstr ""
 "는 것을 기다리는 것과 같은동시성도 구현할 수 있습니다."
 
 #: src/async/tasks.md:16
+#, fuzzy
 msgid "\"127.0.0.1:0\""
-msgstr ""
+msgstr "\"127.0.0.1:2000\""
```
Comment (jiyongp) [inline po/ko.po:19499]: I don't understand how the translation got wrong.

```suggestion
msgstr "\"127.0.0.1:0\""
```

File: po/ko.po [hunk 2]
```diff
@@ -19459,28 +19494,31 @@ msgstr ""
...
 
 #: src/async/tasks.md:17
+#, fuzzy
 msgid "\"listening on port {}\""
-msgstr ""
+msgstr "\"포트 6142에서 수신 대기\""
```
Comment (jiyongp) [inline po/ko.po:19504]:
```suggestion
msgstr "\"포트 {}에서 수신 대기\""
```

File: po/ko.po [hunk 3]
```diff
@@ -19459,28 +19494,31 @@ msgstr ""
 "는 것을 기다리는 것과 같은동시성도 구현할 수 있습니다."
 
 #: src/async/tasks.md:16
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:19497]:
```suggestion
```

File: po/ko.po [hunk 4]
```diff
@@ -19459,28 +19494,31 @@ msgstr ""
...
 msgid "\"127.0.0.1:0\""
-msgstr ""
+msgstr "\"127.0.0.1:2000\""
 
 #: src/async/tasks.md:17
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:19502]:
```suggestion
```

File: po/ko.po [hunk 5]
```diff
@@ -19459,28 +19494,31 @@ msgstr ""
...
 
 #: src/async/tasks.md:22
 msgid "\"connection from {addr:?}\""
-msgstr ""
+msgstr "\"{addr:?}에서 연결\""
 
 #: src/async/tasks.md:25
 msgid "b\"Who are you?\\n\""
-msgstr ""
+msgstr "b'누구세요?\\n\""
```
Comment (jiyongp) [inline po/ko.po:19512]:
```suggestion
msgstr "b\"누구세요?\\n\""
```

File: po/ko.po [hunk 6]
```diff
@@ -19459,28 +19494,31 @@ msgstr ""
...
 
 #: src/async/tasks.md:25 src/async/tasks.md:28 src/async/tasks.md:31
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:19515]:
```suggestion
```

File: po/ko.po [hunk 7]
```diff
@@ -19459,28 +19494,31 @@ msgstr ""
...
 msgid "\"socket error\""
-msgstr ""
+msgstr "\"소켓 오류: {e:?}\""
```
Comment (jiyongp) [inline po/ko.po:19517]:
```suggestion
msgstr "\"소켓 오류\""
```

File: po/ko.po [hunk 8]
```diff
@@ -19768,6 +19809,8 @@ msgid ""
 "An alternative is to pass `&mut future` instead of the future itself, but "
 "this can lead to issues, further discussed in the pinning slide."
 msgstr ""
+"대안은 future 자체 대신 `&mut future`를 전달하는 것입니다. 하지만 이렇게 하"
+"면 문제가 발생할 수 있습니다(고정 슬라이드에서 자세히 설명함)."
```
Comment (jiyongp) [inline po/ko.po:19813]:
```suggestion
"면 문제가 발생할 수 있습니다(Pinning을 다룰 때 자세히 설명할 예정임)."
```

File: po/ko.po [hunk 9]
```diff
@@ -19792,16 +19835,17 @@ msgstr "[실행자 차단](pitfalls/blocking-executor.md)"
 #: src/async/pitfalls.md:8
 #, fuzzy
```
Comment (jiyongp) [inline po/ko.po:19836]:
```suggestion
```

File: po/ko.po [hunk 10]
```diff
@@ -19792,16 +19835,17 @@ msgstr "[실행자 차단](pitfalls/blocking-executor.md)"
...
 msgid "[Pin](pitfalls/pin.md)"
-msgstr "[고정](pitfalls/pin.md)"
+msgstr "[취소](pitfalls/cancellation.md)"
```
Comment (jiyongp) [inline po/ko.po:19838]:
```suggestion
msgstr "[Pin](pitfalls/pin.md)"
```

File: po/ko.po [hunk 11]
```diff
@@ -19792,16 +19835,17 @@ msgstr "[실행자 차단](pitfalls/blocking-executor.md)"
...
 
 #: src/async/pitfalls.md:9
 #, fuzzy
```
Comment (jiyongp) [inline po/ko.po:19841]:
```suggestion
```

File: po/ko.po [hunk 12]
```diff
@@ -19792,16 +19835,17 @@ msgstr "[실행자 차단](pitfalls/blocking-executor.md)"
...
 msgid "[Async Traits](pitfalls/async-traits.md)"
-msgstr "[비동기 트레잇](pitfall/async-traits.md)"
+msgstr "[취소](pitfalls/cancellation.md)"
```
Comment (jiyongp) [inline po/ko.po:19843]:
```suggestion
msgstr "[비동기 트레잇](pitfall/async-traits.md)"
```

File: po/ko.po [hunk 13]
```diff
@@ -19792,16 +19835,17 @@ msgstr "[실행자 차단](pitfalls/blocking-executor.md)"
...
 
 #: src/async/pitfalls.md:10
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:19846]:
```suggestion
```

**Review by jiyongp — APPROVED** [review]

**Pull request merged**